File: regex/Java/Regex3.java
import java.util.regex.*;

class Regex3
{
  public static void main( String args[] )
  {
    String out = new String( "jack\td\therrington" );

    Pattern pat = Pattern.compile( "\\t" );
    out = (pat.matcher( out )).replaceAll( "," );

    System.out.println( out );
  }
}

File: regex/CSharp/regex3.cs
using System;
using System.Text.RegularExpressions;

class Regex3
{
  static void Main()
  {
    Regex r = new Regex( "\t" );
    String str = r.Replace( "jack\td\therrington", "," );
    Console.WriteLine( str );
  }
}
